aws_lambda.py

```diff
@@ -18,7 +18,17 @@
     env_option = {}
     if SENTRY_ENV:
         env_option = {"environment": SENTRY_ENV}
-    sentry_sdk.init(SENTRY_DSN, integrations=[AwsLambdaIntegration()], **env_option)
+    if os.getenv("AWS_LAMBDA_FUNCTION_NAME"):
+        # We're running in AWS. See https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html
+        from sentry_sdk.integrations.aws_lambda import AwsLambdaIntegration
+        integrations = [AwsLambdaIntegration()]
+    elif os.getenv("GOOGLE_CLOUD_PROJECT"):
```

Comment:
Are we sure that we're going to be using one of the "older" runtimes (that provide this environment variable)? Should we also check for [one of these variables](https://cloud.google.com/functions/docs/configuring/env-var#newer_runtimes)? Either that, or we need to tell SRE to export an environment variable (see the note below the section I linked)